csun-metalab/nomi
# NAM-495 -- UI: Fix underlying issue with themes

# Description
Updated theme logic and added loading functionality to main vue's.
  
# Testing
1. Visit all major vue routes and hard refresh to make sure there is a loading screen until the theme is loaded.
2. Visit the settings route; select a theme and make sure it applies properly.
3. Logout and log back in to ensure the theme is properly set.
